docs/en/admin-guide/security/index.rst

```diff
@@ -5,13 +5,15 @@ User Roles
 
 TWW comes with a set of predefined roles for a multi user configuration.
 
-The base installation of TWW defines the following group roles :
+The base installation of TWW defines the following group roles over the entire database cluster:
 
 * **tww_viewer**: has the rights for project consultation, can select entities of TWW, view values lists.
 * **tww_user**: main TWW user, can edit entities (`tww_od` schema), view values lists.
 * **tww_manager**: user of TWW with extended privileges, can edit entities and values lists (`tww_vl` schema).
 * **tww_sysadmin**: superuser of TWW database, can edit TWW system tables (`tww_sys` schema).
 
+Their rights are cascaded to database specific group roles **tww_viewer_db_identifier** etc. if those have been set up. ``db_identifier`` is defined as ``regexp_replace(databasename, "tww_|teksi_", "")`` .
```

Comment:
```suggestion
Their rights are cascaded to database specific group roles **tww_viewer_[db_identifier]** etc. if those have been set up. ``db_identifier`` is defined as ``regexp_replace(databasename, "tww_|teksi_", "")`` .
```

Reply:
regexp_replace(databasename, "tww_|teksi_", "") => this would be the default if we implement this from Python.